README.md

```diff
@@ -6,9 +6,17 @@ Requirements are in `requirements.txt`
 Could be installed with:
 `pip install -r requirements.txt`
 
-## Build 
+## Build
 
 ```
 make html
 ```
 
+## Displaying the site locally
+
+```bash
+cd build/html
+python3 -m http.server
+```
+
+Then, navigate to http://localhost:8000
```

Comment:
Why do you want to serve the page?  I thought browsers would be able to open the file directly

Reply:
Yes I agree you can just open the index.html, but every time you edit the source you need to refresh the html. I actually did not try the local host, maybe it refreshes automatically?